apps/test/solutions/applab/ec_turtle.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
         // room to add tests here
         assert($('#screen1'));
         assert($('#turtleImage'));
-        assert.equal($('#divApplab .screen').length, 1);
+        assert.equal($('#divApplab > .screen').length, 1);
```

Comment:
Not sure how much you want to for tests, but could use `Applab.getScreens()` here and below?